### PR TITLE
feat: laravel 6 support

### PR DESCRIPTION
**Pull request by atymic**

### CODE REVIEW

**Comment by atymic** [comment]
Hey @unicodeveloper 

Any chance you could add me as a contributor on https://github.com/unicodeveloper/jusibe-php-lib so I can keep it up to date?

It's blocking us from laravel 6 support here.

Thanks :)

**Comment by atymic** [comment]
ping @unicodeveloper  :)

**Comment by unicodeveloper** [comment]
Please check the tests that are failing @atymic

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
+  - 7.2
```
Comment (unicodeveloper) [inline .travis.yml:4]: Any particular reason for skipping 7.0 and 7.1?

File: composer.json [hunk 2]
```diff
@@ -12,15 +12,15 @@
         }
     ],
     "require": {
-        "php": ">=5.6.4",
+        "php": ">=7.1",
```
Comment (unicodeveloper) [inline composer.json:15]: It will be great for us not to skip 7.0. A lot of folks got on to 7.0 and they haven't upgraded yet.

**Review by unicodeveloper — COMMENTED** [review]
Thanks for the PR. Please fix and check the failing tests.

File: composer.json (hunk 2 above)
Comment (atymic) [reply composer.json:15]: Fair point, happy to drop back to 7.0 (even though it's out of support)

File: .travis.yml (hunk 1 above)
Comment (atymic) [reply .travis.yml:4]: They are both out of support.

**Comment by atymic** [comment]
> Thanks for the PR. Please fix and check the failing tests.

The tests are failing as per my comment above because your other library doesn't support newer laravel version.

File: composer.json (hunk 2 above)
Comment (unicodeveloper) [reply composer.json:15]: Yeah. Sometime around end of Q1, we can drop it totally.

File: .travis.yml (hunk 1 above)
Comment (unicodeveloper) [reply .travis.yml:4]: Yeah. we can drop it totally next year.

**Comment by unicodeveloper** [comment]
> > Thanks for the PR. Please fix and check the failing tests.
> 
> The tests are failing as per my comment above because your other library doesn't support newer laravel version.

Aha! Didn't even check. Okay, can you update the code with the fix to my comments so I can merge?

**Pull request closed without merging**